src/core/meshcop/network_name.hpp

```diff
@@ -107,6 +107,9 @@ class NameData : private Data<kWithUint8Length>
 class NetworkName : public otNetworkName, public Unequatable<NetworkName>
 {
 public:
+    static constexpr const char *sNetworkNameInit = "OpenThread";
+    static constexpr const char *sDomainNameInit  = "DefaultDomain";
```

Comment:
how about `kNetworkNameInit` (instead of `sNetworkNameInit`) since they are `const`?